src/components/common/Card/Card.jsx

```diff
@@ -47,20 +48,25 @@ export default function Card({ button }) {
     };
     fetchUser();
   }, [user]);
+  const handleButtonClick = () => {
```

Comment:
the function is added but not called anywhere.

Reply:
It is not need to be called ,That i have kept just for debuggging